azerothcore/azerothcore-wotlk
# fix(DB/loot): Highland Fleshstalker drops

## Changes Proposed:
-  Highland fleshstalkers have a bunch of 55+ blues on their loot table that they shouldn't be dropping.
-  These blues are coming from reference loot table 24017
```SELECT * FROM `acore_world`.`reference_loot_template` WHERE ENTRY = 24017;```
-  Removed the reference loot table that was allowing these blues to be dropped from highland fleshstalkers

## Issues Addressed:

- Closes azerothcore#6168
- Closes chromiecraft/chromiecraft#752

## SOURCE:

- https://classic.wowhead.com/npc=2561/highland-fleshstalker#drops;0-2+16-20+1

## Tests Performed:

- Ran the sql script and inspected the resulting database changes.


## How to Test the Changes:


1. Run ```SELECT * FROM `acore_world`.`creature_loot_template` WHERE ENTRY = 2561;``` and ensure that item reference table 24017 is no longer included

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
